spec/support/fixture.rb

```diff
@@ -0,0 +1,32 @@
+begin
+  require 'middleman-core/rack'
+rescue LoadError
```

Comment:
I don't specifically know... I will remove it, and if it ever freaks out in the future we can readd it.

#ITotallyStoleAllThisCodeFromTheInternetYOLO